diff -ruNb a/termcap b/termcap
--- a/termcap	2012-06-10 15:30:37.000000000 +0100
+++ b/termcap	2013-06-02 22:19:53.607392989 +0100
@@ -234,7 +234,7 @@
 # is widely used for a variety of incompatible terminal emulations including
 # color_xterm and rxvt.
 v0|xterm|X11 terminal emulator:\
-	:tc=xterm-new:
+	:tc=xterm-new:kb=^?:
 #	:tc=xterm-r6:
 
 # This fragment is for people who cannot agree on what the backspace key
diff -ruNb a/terminfo b/terminfo
--- a/terminfo	2012-09-05 01:24:08.000000000 +0100
+++ b/terminfo	2013-06-02 22:20:06.036678783 +0100
@@ -1975,3 +1975,4 @@
 # should send.
 xterm+kbs|fragment for backspace key,
 	kbs=^H,
+	kbs=\177,
